spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/workflows.xml

```diff
@@ -0,0 +1,30 @@
+<workflows objectId="druid:bb077xv1376">
```

Comment:
new objects don't have a workflows datastream either, can this be removed?

Reply:
Removed, and tests still pass.